client/src/app/containers/Game/PlayingCard.tsx

```diff
@@ -41,15 +70,14 @@ const PlayingCard: React.FunctionComponent<Props> = ({
     return (
         <div
             onClick={handleClick}
-            className={`w-12 h-16 text-center align-middle inline-block border-2 border-black ${
-                disabled ? 'bg-gray-500' : 'bg-white'
-            } ${useRed ? 'text-red-700' : 'text-black'}`}
-            style={{
-                position: 'relative',
-                top: chosen ? '-10px' : '',
-            }}
+            className={clsx(classes.cardBase, classes.cardSize, {
+                [classes.disabledCard]: disabled,
+                [classes.redCard]: useRed,
+                [classes.blackCard]: !useRed,
+                [classes.selected]: chosen,
+            })}
```

Comment:
Me, predicting Josh's comment :crystal_ball: :
[clsx]() is a cool little (and I mean little; only a few hundred bytes) library that makes it nicer to conditionally apply styles. It takes a variadic list of inputs that can be arrays, strings, and objects; the object syntax means something like this:
```
{
    [className]: someConditionToIncludeThatClassName,
}
```